Content/ReleaseBriefings/release-46.tex

```diff
@@ -0,0 +1,117 @@
+\input{KokkosTutorial_PreTitle}
+\usepackage{tikz}
+\graphicspath{{4_6/figures/}}
+
+%disclaimer for Sandia. uncomment and the whole blob goes away @ b80c116300122
+% \def\sandid{SANDXXXX PE}
+\def\ornlid{THIS WILL NEED TO BE UPDATED AFTER RESOLUTION}
+
+% \title{Performance Portability with Kokkos}
+\title{Kokkos 4.6 Release Briefing}
+
+%BAD misuse of author field
+\author{New Capabilities}
+
+\date{04/10/2025}
+
+\input{KokkosTutorial_PostTitle}
+
+\begin{document}
+
+\begin{frame}
+  \titlepage
+\end{frame}
+
+
+\begin{frame}[fragile]{Outline}
+
+  \textbf{4.6 Release Highlights}
+
+  \begin{itemize}
+    \item{Organizational}
+    \item{Feature Highlights}
+    \item{General Enhancements}
+    \item{Backend updates}
+    \item{Build system updates}
+    \item{Deprecations and other breaking changes}
+    \item{Bug Fixes}
+  \end{itemize}
+
+\end{frame}
+
+\begin{frame}{Find More}
+
+  \textbf{Online Resources}:
+
+  \begin{itemize}
+    \item \url{https://github.com/kokkos}:
+          \begin{itemize}
+            \item Primary Kokkos GitHub Organization
+          \end{itemize}
+    \item \url{https://github.com/kokkos/kokkos-tutorials/wiki/Kokkos-Lecture-Series}:
+          \begin{itemize}
+            \item{Slides, recording and Q\&A for the Full Lectures}
+          \end{itemize}
+    \item \url{https://kokkos.org/kokkos-core-wiki}:
+          \begin{itemize}
+            \item Wiki including API reference
+          \end{itemize}
+    \item \url{https://kokkosteam.slack.com}:
+          \begin{itemize}
+            \item Slack workspace for Kokkos.
+            \item Please join: fastest way to get your questions answered.
+            \item Can whitelist domains, or invite individual people.
+          \end{itemize}
+  \end{itemize}
+
+\end{frame}
+
+\begin{frame}[fragile]{Kokkos Usage}
+  \textbf{Would like to strengthen community bonds and discoverability}
+
+  \vspace{10pt}
+  \textit{List of Applications and Libraries}
+  \begin{itemize}
+    \item Add your app to \url{https://github.com/kokkos/kokkos/issues/1950}
+    \item We are planning to add that to a Kokkos website.
+    \item Helps people discover each other when working on similar things.
+  \end{itemize}
```

Comment:
We might want to finally revisit that and point to https://kokkos.org/community/applications/
@crtrott what do you think?